builtin/providers/aws/resource_aws_spot_instance_request.go

```diff
@@ -194,8 +194,13 @@ func resourceAwsSpotInstanceRequestRead(d *schema.ResourceData, meta interface{}
 			return fmt.Errorf("[ERR] Error reading Spot Instance Data: %s", err)
 		}
 	}
-	d.Set("spot_request_state", *request.State)
-	d.Set("block_duration_minutes", *request.BlockDurationMinutes)
```

Comment:
Good catch, I forgot about that :smile: 

`d.Set` takes care of nil references internally, so we need to do 

```
d.Set("block_duration_minutes", request.BlockDurationMinutes)
```

I'm fixing that now